### ISSUE TITLE
Publish prereleases on conda-forge

### ISSUE THREAD

**Issue by jasongrout** [conda-forge/jupyterlab-feedstock#166]
We had a [discussion on gitter](https://gitter.im/conda-forge/conda-forge.github.io?at=5d518e33beba830fff7dcc67) about publishing prereleases to a `conda-forge/label/jupyterlab-prerelease` channel.

See the discussion for more details, but it looks like we should be able to do this with a `recipe/conda_build_config.yaml` file like [this example](https://github.com/conda-forge/sagelib-feedstock/blob/master/recipe/conda_build_config.yaml):

```yaml
channel_sources:
- conda-forge/label/prerelease-jupyterlab,conda-forge,defaults
channel_targets:
- conda-forge prerelease-jupyterlab
```

Notes:
* Use a package-specific prerelease label: see https://github.com/conda/conda/issues/8752. So `jupyterlab-prerelease` or `prerelease-jupyterlab`, not just `prerelease`. Also, there is no real convention for label name here, so we can make it up. Perhaps `jupyterlab-dev` is better and easier to type? There is some usage of a plain `rc` label (see the issue above).
* Likely we would need to also publish prereleases of `jupyterlab_server` to this same label, if they are in lockstep
* We would probably maintain a parallel prerelease feedstock branch, and keep feedstock master on the latest stable. If we maintain multiple stable versions concurrently, we should probably also branch off for each stable release we concurrently maintain. One caveat is that the autotick bot only looks at the master branch.

**Comment by jasongrout** [comment]
I also said I'd attempt to write up a rough draft of notes about how this works for others.

**Comment by blink1073** [comment]
Time to dust off the bash script I had made for help with conda-forge releases before autotick: https://github.com/blink1073/dotfiles/blob/b04b0291d5e5d4fa766eecf9176f098274d2b02e/bashrc#L100

**Comment by jasongrout** [comment]
> Time to dust off the bash script I had made for help with conda-forge releases before autotick:

So this is sort of a cyborg autotick bot?

**Comment by blink1073** [comment]
Ha!

**Comment by jasongrout** [comment]
Fixed with #172, at least manually for one prerelease.

I'll open another issue for automatically publishing prereleases.
